ups/deploy_testing_distrib.build

```diff
@@ -0,0 +1,2 @@
+@LSST BUILD@ &&
+build_lsst @PRODUCT@ @VERSION@ @REPOVERSION@
```

Comment:
This shouldn't be needed.  I looked at a few of the existing "metapackages" and most of them do have it but at least `qserv_distrib` does not.